azure-pipelines-official.yml

```diff
@@ -317,8 +317,9 @@ stages:
         command: push
         packagesToPush: '$(Build.SourcesDirectory)\artifacts\LanguageServer\*.nupkg'
         allowPackageConflicts: false
+        feedsToUse: select
         nuGetFeedType: external
-        publishFeedCredentials: 'DevDiv - VS package feed'
+        publishFeedCredentials: 'azure-public/vs-impl'
```

Comment:
Per @dibarbet the expectation this is temporary until this all gets deleted as a part of #68504.